capabilities_matrix/_topics/infrastructure_providers.md

```diff
@@ -0,0 +1,65 @@
+| VM / Instance Power Operations                               | Yes            | Yes                          | Yes                                                     | No                                |
+| VM / Instance Retirement                                     | Yes            | Yes                          | Yes                                                     | No                                |
+| Alerts - Real Time                                           | Yes            | No                           | No                                                      | No                                |
+| Alerts - VM Value Changed                                    | Yes            | Yes                          | No                                                      | No                                |
+| Alerts - VM Reconfigured                                     | Yes            | Yes                          | No                                                      | No                                |
+| Integrate with Service Catalogs                              | Yes            | Yes                          | Yes                                                     | No                                |
+| Virtual Machine Reconfiguration                              | Yes            | Yes                          | No                                                      | No                                |
+| Automation Work Flows                                        | Yes            | Yes                          | Yes                                                     | No                                |
+| Provision VM/Instance using PXE                              | Yes            | Yes                          | No                                                      | No                                |
+| Provision VM/Instance using ISO                              | No             | Yes                          | No                                                      | No                                |
+| Provision from Template/Image to VM/Instance                 | Yes            | Yes                          | Yes                                                     | No                                |
+| Provision from VM/Instance to Template/Image                 | Yes            | Yes                          | No                                                      | No                                |
+| Clone from VM/Instance to VM/Instance                        | Yes            | No                           | No                                                      | No                                |
+| Disk Addition to VM/Instance                                 | Yes            | No                           | No                                                      | No                                |
```

Comment:
Ovirt does support disk add/remove now

Reply:
This table is also missing a "Network Interface Add/Remove to VM/Instance" feature, which is supported by VMware and Ovirt